x-pack/filebeat/module/gsuite/user_accounts/config/pipeline.js

```diff
@@ -0,0 +1,24 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+var userAccounts = (function () {
+    var processor = require("processor");
+
+    var categorizeEvent = function(evt) {
+        evt.Put("event.type", "change");
+        evt.Put("event.category", "authentication");
```

Comment:
I'm wondering if event.category should be "iam", are these login/authentication events or user modification events?

Reply:
True 👍